server/utils/mail_templates.py

```
@@ -210,6 +210,87 @@ def send_verification_email(to_email, user_name, verify_link):
+        <div class="content">
+            <p>Dear {user_name},</p>
+            <p>We received a request to reset your EcoVibe account password.
+            You can reset it by clicking the button below:</p>
+
+            <p style="text-align:center;">
+                <a href="{reset_link}" class="button">Reset Password</a>
+            </p>
+
+            <p>This link will expire in 30 minutes.</p>
+
+            <p>If the button above doesn’t work, copy and paste this link
```

Ruff (0.13.1)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)